Nonprofit-Exchange-Hub/web-app
# home page CTA cards

## Dev Summary

[[Mocks](link)](https://www.figma.com/file/KE2Sw6ob07agN25RmeNuFg/GiveCycle-wireframes?node-id=4076%3A12228)


- Created a new component CTACard.tsx within the components directory. 
- Imported CTACard.tsx into Home.tsx to display on the landing page above the recent needs section and below the hero image section. 
- Imported necessary elements into the CTACard component as well as placeholder text. 
- Worked to follow the wireframe spacing as outlined, some sizes such as the arrow were adjusted as the wireframe had them as too small in comparison to the rest of the component. 
- Mouseover effect does not match intended mock, which is more of a transition where the arrow grows. I was unsure how to match the intended effect using Material UI. 
- Headline responsiveness needs work when keeping mobile devices in mind, the rest of the component should be responsive as intended. 


## Test Plan


repro steps:

1. localhost:3000 
2. click X button
expected: 


1. localhost:3000
2. open homepage and navigate to section below hero image. 
expected: You should see 3 CTA cards with placeholder text. I used a placeholder image for the circle in the bottom left and the arrow is on the bottom right of the card. 

## Resources

- [stackoverflow is cool](stackoverflow.com)
- MUI official documentation for: Grid, Avatar, Card, CardContent, IconButton, Typography, MUI Icons, makeStyles from MUI styles.
